core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -825,11 +825,11 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    *
    * @note Small files are preferred, large file is also allowable, but may cause bad performance.
    *
-   * @param minPartitions A suggestion value of the minimal splitting number for input data.
+   * @param maxPartitions A suggestion value of the maximal splitting number for input data.
    */
   def wholeTextFiles(
       path: String,
-      minPartitions: Int = defaultMinPartitions): RDD[(String, String)] = withScope {
+      maxPartitions: Int = defaultMinPartitions): RDD[(String, String)] = withScope {
```

Comment:
it's strange that this still defaults to `defaultMinPartitions`. Does that need to be fixed as well?